website/meta/universe.json

```diff
@@ -3238,34 +3238,62 @@
             "github": "babylonhealth/hmrb",
             "pip": "hmrb",
             "code_example": [
-                "import spacy # __version__ 3.0+",
+                "import spacy",
                 "from hmrb.core import SpacyCore",
```

Comment:
```suggestion
```
Looks like this import can be removed?

Reply:
It's is important for the `nlp.add_pipe(\"hmrb\", config=conf)` to find the factory